selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/RenameOrganizationTest.java

```diff
@@ -38,34 +38,32 @@
  * @author Ann Shumilova
  */
 public class RenameOrganizationTest {
-  private static final Logger LOG = LoggerFactory.getLogger(RenameOrganizationTest.class);
+  private static final String PARENT_ORG_NAME = generate("organization", 5);
+  private static final String CHILD_ORG_NAME = generate("organization", 5);
+  private static final String NEW_PARENT_ORG_NAME = generate("organization", 5);
```

Comment:
It's better to add **"new-parent"** prefix to the name to simplify test result analysis.